python/plugins/processing/algs/help/qgis.yaml

```diff
@@ -402,12 +402,14 @@ qgis:polygoncentroids: >
 
   NOTE: This algorithm is deprecated and the generic "centroids" algorithm (which works for line and multi geometry layers) should be used instead.
 
-
 qgis:polygonfromlayerextent: >
```

Comment:
Why not renaming this one as `polygonfromvectorextent` to be consistent with the new one?

Reply:
The id isn't shown anywhere - I've kept them as their 2.x versions just to (very slightly) minimise porting efforts for plugin authors